frontend/src/individual_form.tsx

```diff
@@ -351,25 +353,19 @@ export function IndividualForm({
                     </div>
                   </>
                 ) : formAction == FormAction.handleCertificate ? (
+                  
                   <div className="flexRow">
-                    <KeyboardDatePicker
-                      required
-                      error={birthDateError}
-                      autoOk
-                      variant="inline"
+                  <Tooltip title="Är datumet fel vänligen ändra i parningstillfället" placement="right">  
```

Comment:
This tooltip is only visible on hover over the input field. There is no indication that there is more information, like a small "i" in a circle or similar. If possible, it would be good to add that to improve the UI.

Reply:
How can I do that?